python/pyspark/sql/protobuf/functions.py

```diff
@@ -232,6 +274,12 @@ def to_protobuf(
     return Column(jc)
 
 
+def _read_descriptor_set_file(filePath: str) -> bytes:
+    # TODO: Throw structured errors like "PROTOBUF_DESCRIPTOR_FILE_NOT_FOUND" etc.
```

Comment:
and should file TODO'ed JIRA such as `TODO(SPARK-XXXXX)`. It's a bit trouble some to file for every todo but we found that some of todos are just left out, forgotten, and nobody remembers what it was :-).

Reply:
Agree. Added jira ticket and updated the comment.
Can we throw structured error now in Python? In that case, I can actually fix it in this PR itself. 
Please point to an example.